### ISSUE TITLE
Rename `_CHALKS` to `_ARTIFACT_INFO`

### ISSUE THREAD

**Issue by nettrino** [crashappsec/chalk#202]
The `_CHALKS` key contains metadata about artifacts, some of which are part of the chalkmarks (those without `_`) and some of which are part of the per-artifact runtime info. They do _no_ correspond to chalkmarks. We should probably rename them to prevent any confusion on behalf of the users

**Comment by viega** [comment]
Agreed this is an improvement, though I think we could leave _CHALKS as an alternate name in there for a while if people already are expecting it?  How to communicate about deprecation...

**Comment by nettrino** [comment]
@viega agreed - we should also decide what our approach around deprecating keys is - for instance do we keep keys around for the next N major revisions? How do we declare them as deprecated in con4m? Tracking deprecation discussion in  https://github.com/crashappsec/chalk/discussions/204
